common/changes/@itwin/presentation-common/Presentation-removeLocalization_2022-08-01-07-12.json

```diff
@@ -2,9 +2,9 @@
   "changes": [
     {
       "packageName": "@itwin/presentation-common",
-      "comment": "Move localizationhelper",
+      "comment": "Added 'getLocalizedStringEN' function for localizing common Presentation strings to english",
```

Comment:
`getLocalizedStringEN` is `@alpha`, so it shouldn't be referenced from the docs / changelogs. 

If we're planning for our consumers to call it (which I'm not sure they will), then:
- the function should be changed to `@beta`
- it should be referenced as code rather than in quotes
- the word "English" should start with upper case

I'd first think if it shouldn't be `@internal`, though. In that case this message could be left empty.